Update adding-model.md

chpong opened the pull request:
Remove extra wording "these services" in MVC tutorial.



<!--
# Instructions

When creating a new PR, please reference the issue number if there is one:

Fixes #Issue_Number

The "Fixes #nnn" syntax in the PR description allows GitHub to automatically close the issue when this PR is merged.

NOTE: This is a comment; please type your descriptions above or below it.
-->

guardrex commented:
Thanks @chpong ... I updated the other spots. These markdown files often have multiple copies of the content, one for each of our covered versions. @Rick-Anderson will be along in a bit to signoff and merge this. Thanks again, @chpong! 🚀

chpong commented:
> Thanks @chpong ... I updated the other spots. These markdown files often have multiple copies of the content, one for each of our covered versions. @Rick-Anderson will be along in a bit to signoff and merge this. Thanks again, @chpong! 🚀

Thanks @guardrex. I updated the Razor Pages version earlier in #23346. Were there other spots that I can check in the future?
